app/grandchallenge/algorithms/migrations/0065_create_interfaces_for_jobs.py

```diff
@@ -0,0 +1,28 @@
+from django.db import migrations
+
+
+def add_algorithm_interfaces_to_jobs(apps, _schema_editor):
+    Algorithm = apps.get_model("algorithms", "Algorithm")  # noqa: N806
+    Job = apps.get_model("algorithms", "Job")  # noqa: N806
+
+    for algorithm in Algorithm.objects.prefetch_related("interfaces").all():
+        default_interface = algorithm.interfaces.get(
+            algorithmalgorithminterface__is_default=True
+        )
+        jobs = Job.objects.filter(algorithm_image__algorithm=algorithm)

```

Comment:
Note to self: using the current `default_interface` of the algorithm for all its past jobs might be incorrect for some jobs. Editors might have changed the interfaces between jobs. So I guess the better approach would be to determine the job's interface based on its inputs. When this piece of code runs, each algorithm will have only 1 interface though. So then we should also add the job's interface to the list of interfaces for the algorithm if it's not listed there yet.